src/BaseHandlerTrait.php

```diff
@@ -21,54 +18,31 @@
 trait BaseHandlerTrait
 {
     /**
-     * @return Decoder
-     *
-     * @deprecated 4.5.0 Will be removed without equivalent functionality to replace it
-     *
-     * No longer needed as of v3 of webauthn-lib the Decoder is now created within both:
-     * AttestationObjectLoader::__construct()
-     * AuthenticatorAssertionResponseValidator::__construct()
-     */
-    protected function getDecoder(): Decoder
-    {
-        return Deprecation::withNoReplacement(function () {
-            Deprecation::notice('4.5.0', 'Will be removed without equivalent functionality to replace it');
-            return new Decoder(new TagManager(), new OtherObjectManager());
-        });
-    }
-
-    /**
-     * @param Decoder $decoder
      * @return AttestationStatementSupportManager
      */
-    protected function getAttestationStatementSupportManager(Decoder $decoder): AttestationStatementSupportManager
+    protected function getAttestationStatementSupportManager(): AttestationStatementSupportManager
     {
         $manager = new AttestationStatementSupportManager();
         $manager->add(new NoneAttestationStatementSupport());
-        $manager->add(new FidoU2FAttestationStatementSupport($decoder));
```

Comment:
I'm unsure if we're supposed to be removing this here

Reply:
DONE